packages/palette/src/helpers/media.ts

```diff
@@ -22,7 +22,7 @@ export const media: Media = Object.entries(breakpoints).reduce(
     return {
       ...accumulator,
       [label]: (strings, ...args) => css`
-        @media (max-width: ${value}px) {
+        @media (max-width: ${value}) {
```

Comment:
In force, would look around for this `media` import (there's two afaik, one for old way of doing things and this), as well as `breakpoint`, etc.